pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -139,12 +145,56 @@ func makePodByResourceListWithManyContainers(resources *v1.ResourceList, contain
 func extractResources(zone topologyv1alpha1.Zone) v1.ResourceList {
 	res := make(v1.ResourceList)
 	for _, resInfo := range zone.Resources {
-		klog.V(5).InfoS("Extract resources for zone", "resName", resInfo.Name, "resAvailable", resInfo.Available)
 		res[v1.ResourceName(resInfo.Name)] = resInfo.Available
 	}
 	return res
 }
 
+func logNumaNodes(desc string, nodes NUMANodeList) {
+	if !klog.V(6).Enabled() {
+		// avoid even the trivial range over nodes
+		return
+	}
+	for _, numaNode := range nodes {
+		numaIdent := fmt.Sprintf("node-%d", numaNode.NUMAID)
+		klog.V(6).InfoS(desc, resourceListToLoggable(numaIdent, numaNode.Resources)...)
+	}
+}
+
+func resourceListToLoggable(ident string, resources v1.ResourceList) []interface{} {
+	// count pairs + ident pair
+	items := make([]interface{}, (len(resources)+1)*2)
+	items[0] = "ident"
+	items[1] = ident
+
+	idx := 0
+	resNames := make([]string, len(resources))
+	for resName := range resources {
+		resNames[idx] = string(resName)
+		idx++
+	}
+	sort.Strings(resNames)
+
+	itemIdx := 2
+	for _, resName := range resNames {
+		qty := resources[v1.ResourceName(resName)]
+		items[itemIdx] = resName
+		resVal, _ := qty.AsInt64()
+		if needsHumanization(resName) {
```

Comment:
Maybe capture the reason why we are checking for memory and hugepages and handling differently here as a comment here to explain that we expect that the CPUs are already in human readable form as they are expected to be integral.

Reply:
sure thing